mlir/lib/Dialect/Tensor/IR/TensorOps.cpp

```diff
@@ -579,11 +579,30 @@ struct DimOfCastOp : public OpRewritePattern<DimOp> {
     return success();
   }
 };
+
+struct DimOfDestStyleOp : public OpRewritePattern<DimOp> {
```

Comment:
Can you document the pattern?

Reply:
Thank you for the approval!  I added a brief comment: "Fold dim of a destination passing style op into the dim of the corresponding init."   The updated PR should  be running CI now.